.travis.yml

```diff
@@ -0,0 +1,93 @@
+{
+  "addons": {
+    "apt": {
+      "packages": [
+        "libsnappy-dev"
+      ]
+    }
+  },
+  "install": [
+    "pip install tox"
+  ],
+  "language": "python",
+  "matrix": {
+    "include": [
+      {
+        "env": "TOXENV=docs",
+        "python": "2.7"
+      },
+      {
+        "env": "TOXENV=py27-int KAFKA_VERSION=0.8.0",
```

Comment:
Drop support for 0.8.0.  It's ancient, and the integration tests (some at least) fail.